types/locks/locks-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as locks from "locks";
```

Comment:
Since node-style exports are used, `import locks = require("locks");` might be preferable.

Reply:
and done!